extensions/kogito/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,10 +1,14 @@
 ---
 name: "Kogito"
+description: "Add business automation capabilities - processes and rules with Kogito (a toolkit that originates from projects Drools and jBPM)."
```

Comment:
Usually we put the description in the pom.xml. Can you update https://github.com/quarkusio/quarkus/blob/master/extensions/kogito/runtime/pom.xml#L14 instead?

The keywords are fine

Reply:
Sorry to not have it documented somewhere, but we're working on this :wink:

Reply:
description moved to pom.xml